build.gradle

```diff
@@ -322,7 +324,7 @@ task all(dependsOn: ['build', 'slowtest', 'javadoc', 'buildNsis', 'allReports'])
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '3.5'
+    gradleVersion = '4.0'
```

Comment:
This requires additional testing. Its likely okay but actually doesn't do anything on its own without regenerating the wrapper and upgrading gradle itself.